### DISCUSSION TITLE
Final Issues

### DISCUSSION THREAD

**Discussion by gregoryspragginsjr** [ColbyCommunications/colby-college-theme#47] [General]
Hey @bwaltz it looks like you've grabbed a few of these issues in my absence today; thats great. I wanted to make sure we don't overlap as I continue to close on the remaining items. It looks like you guys do indeed want the `Accordion` in some capacity, which is what I intend to work on now. Let me know what you're on, and I continue in whichever direction you all need. Thanks.

**Comment by bwaltz** [comment]
@gregoryspragginsjr yeah if you could do the accordions, that'd be great!

**Comment by bwaltz** [comment]
@gregoryspragginsjr I think we have a handle on the remaining issues. How are the animations coming?

**Comment by gregoryspragginsjr** [comment]
@bwaltz Hey Brandon, thats great! We wanted to make sure you guys were good with the more pertinent issues and content entry before jumping into more animations. Steve and I will be regrouping on direction, and you should see a commit from me likely early next week with visuals.

**Comment by gregoryspragginsjr** [comment]
@bwaltz I've pushed a new commit here that should cover some fundamental animation integrations: https://github.com/ColbyCommunications/colby-college-theme/commit/3e27a7ce58d4ae17a82c92ca850ea02c3226b63a

Let us know your thoughts. We'll likely pepper in a few more whilst handling your requested enhancement on the image grid at: https://github.com/ColbyCommunications/colby-college-theme/issues/50

Keep an eye out for that commit in a few hours.
